terrahelp/mask.go

```diff
@@ -53,8 +54,8 @@ const (
 	ThMaskChar    = "*"
 	ThMaskCharNum = 6
 
-	ThPrev2CurrPattern        = "\"(.+)\"\\s*=>\\s*\"(\\%s*)\""
-	ThPrev2CurrReplacePattern = "\"%s\" => \"%s\""
+	ThPrev2CurrPattern        = "= \"(.+)\"\\s*->\\s*\"(\\%s*)\""
+	ThPrev2CurrReplacePattern = "= \"%s\" -> \"%s\""
```

Comment:
why is the impact of this change in previous terraform versions, does it mean that this now requires a new version?

Reply:
Yes this would only work within Terraform 0.12.x

Reply:
It might actually be better to have a flag (which by default deals with HCL2 / 0.12 format) but can be set to still deal with the previous format as well

Reply:
The latest commit to the branch allows TF 0.11 console output to be processed by setting a flag called enable011.  Flag name it open to better suggestions.  This switches the pattern back to the previous version that is in master.  The older tests have been added to confirm.